### PR TITLE
Cleanup and unify WorldShaper message formatting

### PR DESCRIPTION
**Pull request by CodedStingray**

### CODE REVIEW

File: src/main/java/net/codedstingray/worldshaper/chat/ChatMessageFormatter.java [hunk 1]
```diff
@@ -29,43 +26,43 @@
...
+        return accent(MessageLevel.INFO, message);
     }
 
-    /**
-     * Formats a message to the WorldShaper warning message format.
-     * @param message The warning message
-     */
-    public static String worldShaperWarningMessage(String message) {
-        return WORLDSHAPER_MESSAGE_PREFIX + WARNING_COLOR + message;
+    public static String accent(MessageLevel messageLevel, String message) {
```
Comment (ZidArgs) [inline src/main/java/net/codedstingray/worldshaper/chat/ChatMessageFormatter.java:61]: missing doc

File: src/main/java/net/codedstingray/worldshaper/chat/ChatMessageFormatter.java [hunk 2]
```diff
@@ -29,43 +26,43 @@
...
+    |* Utilities *|
+    \* ========= */
 
-    /**
-     * Formats a message to the WorldShaper message format.
-     * @param message The message
-     */
-    public static String worldShaperMessage(String message) {
-        return WORLDSHAPER_MESSAGE_PREFIX + message;
+    public static String accent(String message) {
```
Comment (ZidArgs) [inline src/main/java/net/codedstingray/worldshaper/chat/ChatMessageFormatter.java:49]: missing doc

File: src/main/java/net/codedstingray/worldshaper/chat/ChatMessageFormatter.java [hunk 3]
```diff
@@ -82,78 +79,287 @@ public static String groupedMessages(String header, String... messages) {
...
+        private final List<Message> messages;
+
+        private MessageLevel messageLevel = MessageLevel.INFO;
+        private boolean prependMessagePrefix = true;
+
+        private WorldShaperMessageBuilder() {
+            this.messages = new LinkedList<>();
+        }
+
+        public WorldShaperMessageBuilder withMessageLevel(MessageLevel messageLevel) {
```
Comment (ZidArgs) [inline src/main/java/net/codedstingray/worldshaper/chat/ChatMessageFormatter.java:156]: missing doc

File: src/main/java/net/codedstingray/worldshaper/chat/ChatMessageFormatter.java [hunk 4]
```diff
@@ -82,78 +79,287 @@ public static String groupedMessages(String header, String... messages) {
...
+            this.messageLevel = messageLevel;
+            return this;
+        }
+
+        public WorldShaperMessageBuilder withMessagePrefix(boolean prependMessagePrefix) {
```
Comment (ZidArgs) [inline src/main/java/net/codedstingray/worldshaper/chat/ChatMessageFormatter.java:167]: missing doc

File: src/main/java/net/codedstingray/worldshaper/chat/WorldShaperMessages.java [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+    public static String positionSetMessage(int index, Vector3i position, boolean changed) {
+        return messageBuilder(MessageLevel.INFO, true)
+                .t("Position ").a(index + 1)
+                .t(changed ? " set to " : " was already at ")
+                .t(WorldShaperObjectFormatter.vectorToString(position))
+                .t(".")
+                .build();
+    }
+
+    public static String positionRemovedMessage(int index, boolean changed) {
```
Comment (ZidArgs) [inline src/main/java/net/codedstingray/worldshaper/chat/WorldShaperMessages.java:58]: missing doc

File: src/main/java/net/codedstingray/worldshaper/chat/WorldShaperMessages.java [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+        return messageBuilder(MessageLevel.INFO, true)
+                .t("Control position ").a(index + 1)
+                .t(changed ? " removed." : " was already not set.")
+                .build();
+    }
+
+    public static String playerJoinMessage() {
```
Comment (ZidArgs) [inline src/main/java/net/codedstingray/worldshaper/chat/WorldShaperMessages.java:70]: missing doc

File: src/main/java/net/codedstingray/worldshaper/chat/WorldShaperMessages.java [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+        List<String> messages = new LinkedList<>();
+        messages.add("Using " + accent("WorldShaper") + " version " + accent(WorldShaperManifest.PLUGIN_VERSION) + ".");
+        messages.add("");
+        messages.add("Created by " + accent("CreativeConduit"));
+        messages.add("Join our Discord at " + accent(TextColor.UNDERLINE + "https://discord.gg/BDbVGBmCY7"));
+
+        return ChatMessageFormatter.groupedMessages("WorldShaper", messages);
+    }
+
+    public static String worldShaperInfoMessage() {
```
Comment (ZidArgs) [inline src/main/java/net/codedstingray/worldshaper/chat/WorldShaperMessages.java:85]: missing doc

**Review by ZidArgs — CHANGES_REQUESTED** [review]

File: src/main/java/net/codedstingray/worldshaper/chat/ChatMessageFormatter.java (hunk 1 above)
Comment (CodedStingray) [reply src/main/java/net/codedstingray/worldshaper/chat/ChatMessageFormatter.java:61]: added

File: src/main/java/net/codedstingray/worldshaper/chat/ChatMessageFormatter.java (hunk 2 above)
Comment (CodedStingray) [reply src/main/java/net/codedstingray/worldshaper/chat/ChatMessageFormatter.java:49]: added

File: src/main/java/net/codedstingray/worldshaper/chat/ChatMessageFormatter.java (hunk 3 above)
Comment (CodedStingray) [reply src/main/java/net/codedstingray/worldshaper/chat/ChatMessageFormatter.java:156]: added

File: src/main/java/net/codedstingray/worldshaper/chat/ChatMessageFormatter.java (hunk 4 above)
Comment (CodedStingray) [reply src/main/java/net/codedstingray/worldshaper/chat/ChatMessageFormatter.java:167]: added

File: src/main/java/net/codedstingray/worldshaper/chat/WorldShaperMessages.java (hunk 5 above)
Comment (CodedStingray) [reply src/main/java/net/codedstingray/worldshaper/chat/WorldShaperMessages.java:58]: added

File: src/main/java/net/codedstingray/worldshaper/chat/WorldShaperMessages.java (hunk 6 above)
Comment (CodedStingray) [reply src/main/java/net/codedstingray/worldshaper/chat/WorldShaperMessages.java:70]: added

File: src/main/java/net/codedstingray/worldshaper/chat/WorldShaperMessages.java (hunk 7 above)
Comment (CodedStingray) [reply src/main/java/net/codedstingray/worldshaper/chat/WorldShaperMessages.java:85]: added

**Review by ZidArgs — APPROVED** [review]

**Pull request merged**